cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -419,6 +432,14 @@ FUNCTION(DEAL_II_ADD_TEST _category _test_name _comparison_file)
       # And finally define the test:
       #
 
+      IF("${_expect}" STREQUAL "PERFORMANCE")
+        IF(ENABLE_PERFORMANCE_TESTS)
```

Comment:
Can you combine these two `IF` statements into one with `AND`? I think that would make it easier to read, in particular if you list the second condition first.